python/cudf/cudf/core/dtypes.py

```diff
@@ -268,6 +268,19 @@ def deserialize(cls, header: dict, frames: list):
 
         return cls(element_type=element_type)
 
+    @property
+    def nesting_levels(self):
```

Comment:
Should this be a public property? I think we should avoid introducing a public API here unless it's necessary.
```suggestion
    def _nesting_levels(self):
```

Reply:
Removed this property